resources/pgp-keys-map.list

```diff
@@ -312,6 +312,7 @@ net.sf.jtidy                    = 0x4F752A9878E937049EF3FBA7D4012DDA1F1F0F82
 
 net.sf.saxon                    = 0xD9AA7402BA75F78720D975211D185615D0A84648
 
+net.sourceforge.barbecue:barbecue:1.5-beta1 = noSig
```

Comment:
There is only this version
https://repo1.maven.org/maven2/net/sourceforge/barbecue/barbecue/
so specific versions range here is not need now

Reply:
I have rebased and removed the version range.